### ISSUE TITLE
Performance Tweaks

### ISSUE THREAD

**Issue by deniaz** [namics/generator-nitro#44]
Things that (I think) are too slow:
- `gulp develop`
- `yo nitro` (`npm`? `bower`?)
- imageoptimizer optimizes every time, needs to be run once only (per image)
- ...

At one point we could maybe have a look at one or two of these points..

**Comment by deniaz** [comment]
https://www.npmjs.com/package/npm-cache

**Comment by ernscht** [comment]
an offline dependency manager, cache or proxy would be very useful. 
Does anyone have experience with this?

the other ones:
* image optim should run only once per image (for a long time ;-)
* gulp develop does not include npm or bower installs, so it's quiet fast now

**Comment by ernscht** [comment]
I close this performance tweaks.
The dependency manager idea is moved to a new issue: #78
